specifyweb/frontend/js_src/lib/components/FormEditor/spec.ts

```diff
@@ -48,125 +48,86 @@ export type ViewSets = Omit<RawViewSets, 'viewDefs'> & {
   >;
 };
 
-const tablesWithNoBusRulesIn6 = new Set([
-  'AccessionAttachment',
-  'AccessionCitation',
-  'AddressOfRecord',
-  'AgentAttachment',
-  'AgentGeography',
-  'AgentIdentifier',
-  'AgentSpecialty',
-  'AgentVariant',
-  'AttachmentImageAttribute',
-  'AttachmentMetadata',
-  'AttachmentTag',
-  'AttributeDef',
-  'AutonumberingScheme',
-  'BorrowAgent',
-  'BorrowAttachment',
-  'BorrowMaterial',
-  'BorrowReturnMaterial',
-  'CollectingEventAttachment',
-  'CollectingEventAttr',
-  'CollectingTripAttachment',
-  'CollectingTripAttribute',
-  'CollectionObjectAttachment',
-  'CollectionObjectAttr',
-  'CollectionObjectAttribute',
-  'CollectionObjectCitation',
-  'CollectionObjectProperty',
-  'CollectionRelType',
-  'CollectionRelationship',
-  'CommonNameTx',
-  'CommonNameTxCitation',
-  'ConservDescriptionAttachment',
-  'ConservEventAttachment',
-  'DnaPrimer',
-  'DnaSequenceAttachment',
-  'DnaSequencingRunAttachment',
-  'DnaSequencingRunCitation',
-  'DeaccessionAttachment',
-  'DeterminationCitation',
-  'DisposalAttachment',
-  'ExchangeInAttachment',
-  'ExchangeInPrep',
-  'ExchangeOutAttachment',
-  'ExchangeOutPrep',
-  'Exsiccata',
-  'ExsiccataItem',
-  'FieldNotebookAttachment',
-  'FieldNotebookPageAttachment',
-  'FieldNotebookPageSetAttachment',
-  'GeographyTreeDef',
-  'GeographyTreeDefItem',
-  'GeologicTimePeriodTreeDef',
-  'GeologicTimePeriodTreeDefItem',
-  'GiftAgent',
-  'GiftAttachment',
-  'InstitutionNetwork',
-  'LatLonPolygon',
-  'LatLonPolygonPnt',
-  'LithostratTreeDef',
-  'LithostratTreeDefItem',
-  'LoanAgent',
-  'LoanAttachment',
-  'LocalityAttachment',
-  'LocalityCitation',
-  'LocalityNameAlias',
-  'MaterialSample',
-  'MorphbankView',
-  'OtherIdentifier',
-  'PermitAttachment',
-  'PicklistItem',
-  'PreparationAttachment',
-  'PreparationAttr',
-  'PreparationAttribute',
-  'PreparationProperty',
-  'Project',
-  'RecordSet',
-  'RecordSetItem',
-  'ReferenceWorkAttachment',
-  'RepositoryAgreementAttachment',
-  'SpAppResource',
-  'SpAppResourceData',
-  'SpAppResourceDir',
-  'SpAuditLog',
-  'SpAuditLogField',
-  'SpExportSchema',
-  'SpExportSchemaItem',
-  'SpExportSchemaItemMapping',
-  'SpExportSchemaMapping',
-  'SpFieldValueDefault',
-  'SpLocaleContainer',
-  'SpLocaleContainerItem',
-  'SpLocaleItems',
-  'SpPermission',
-  'SpPrincipal',
-  'SpQuery',
-  'SpQueryField',
-  'SpReport',
-  'SpSymbiotaInstance',
-  'SpTaskSemaphore',
-  'SpVersion',
-  'SpViewSetObj',
-  'SpVisualQuery',
-  'StorageAttachment',
-  'StorageTreeDef',
-  'StorageTreeDefItem',
-  'TaxonAttachment',
-  'TaxonAttribute',
-  'TaxonCitation',
-  'TaxonTreeDef',
-  'TaxonTreeDefItem',
-  'TreatmentEventAttachment',
-  'VoucherRelationship',
-  'Workbench',
-  'WorkbenchDataItem',
-  'WorkbenchRow',
-  'WorkbenchRowExportedRelationship',
-  'WorkbenchRowImage',
-  'WorkbenchTemplate',
-  'WorkbenchTemplateMappingItem',
+const tablesWithBusRulesIn6 = new Set([
+  'AccessionAgentBusRules',
```

Comment:
Each of these has `BusRules` at the end, where it previously did not. I expect it to behave incorrectly after this change, but I can review once it has been adjusted or my fears have been alleviated! 😄